demo/src/tests/mod.rs

```diff
@@ -265,7 +265,16 @@ pub fn print_env() -> Result<(), ()> {
 
 pub fn hello() -> Result<(), ()> {
 	println!("Hello, world!");
-
+	println!("Привет мир!");
+	println!("こんにちは世界");
```

Comment:
Naive question. What does this text mean?

Reply:
I used Google translate to translate "Hello World" into multiple languages.

Reply:
I was thinking it, I just didn't realize it.